packages/ibm-products/src/components/Card/Card.js

```diff
@@ -211,7 +229,7 @@ export let Card = forwardRef(
       primaryButtonText,
       primaryButtonDisabled,
       description,
-      hasActions: hasActions,
+      hasActions: hasActions && actionsPlacement === 'top',
```

Comment:
just for consistency sake i think you could move this near line 93 where `hasFooterActions` is set and just set something like `hasHeaderActions`